apps/backend/src/libs/types/service.type.ts

```diff
@@ -1,6 +1,6 @@
 type Service<T = unknown> = {
 	create(payload: unknown): Promise<T>;
-	delete(id: number): Promise<boolean>;
+	delete(id: number, payload: unknown): Promise<boolean>;
```

Comment:
```suggestion
	delete(id: number): Promise<boolean>;
```
you can not change this implement type

Reply:
For delete in service we need for example current user info to check whether user is allowed to delete only his comments

Reply:
do it with a separate call to db for now